README.md

```diff
@@ -144,7 +144,7 @@ const myApp = new k8s.yaml.ConfigFile("app", {
 
 This example creates an EKS cluster with [`pulumi/eks`](https://github.com/pulumi/pulumi-eks),
 and then deploys a Helm chart from the stable repo using the 
-`kubeconfig` credentials from the cluster's [Pulumi provider](https://www.pulumi.com/docs/reference/programming-model/#providers).
+`kubeconfig` credentials from the cluster's Pulumi provider.
```

Comment:
i could not figure out where this content lives now, feel free to let me know if you know and ill update the link